app/src/main/java/agency/five/codebase/android/movieapp/data/di/DataModule.kt

```diff
@@ -0,0 +1,14 @@
+package agency.five.codebase.android.movieapp.data.di
+
+import agency.five.codebase.android.movieapp.data.repository.FakeMovieRepository
+import agency.five.codebase.android.movieapp.data.repository.MovieRepository
+import kotlinx.coroutines.Dispatchers
+
+
+import org.koin.dsl.module
+
+val dataModule = module {
+    single<MovieRepository> {
+        FakeMovieRepository(ioDispatcher = Dispatchers.IO)
```

Comment:
create a concurrencyModule and provide dispatcher IO. Also, provide Dispatchers.Main

to provide two different implementations of some interface, use a named qualifier

https://insert-koin.io/docs/reference/koin-core/definitions#definition-naming--default-bindings